.github/workflows/ci-linux.yml

```diff
@@ -174,7 +174,6 @@ jobs:
         cmake --build ${GITHUB_WORKSPACE}/build
 
   test-utility:
-    needs: [build-gcc, build-clang]
```

Comment:
I've noticed that sometimes jobs in the build matrix fail due to issues not related to nanodbc. For example, sometimes the `actions/checkout@v3` job itself will fail with an error like `libc.so.6: version GLIBC_2.28 not found`. When this happens we skip running all of our tests, because the tests need the _entire_ suite of builds to pass. This gives us an incomplete view of any issues a given PR might introduce. We should run the tests even if some of the build jobs fail.